docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -47,34 +92,61 @@ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
     -Dextensions="spring-di"
 cd spring-di-quickstart
 ----
+////
+
+//Alternative 2: you have a project and need to add the required extension.
+[id="setting-up-your-spring-di-maven-project_{context}"]
+== Setting up your Spring DI example Maven project
 
-This command generates a Maven project with a REST endpoint and imports the `spring-di` extension.
+[role="_abstract"]
+Add the required {ProductName} extension as a dependency of your Maven project.
 
-If you already have your Quarkus project configured, you can add the `spring-di` extension
-to your project by running the following command in your project base directory:
+.Procedure
 
+. Navigate to the the top-level directory of your {ProductName} Maven project:
++
+[source,bash,options="nowrap",subs="+quotes,attributes+"]
+----
+cd <name-of-your-project-directory>
+----
+
+. Execute the following command to add the `quarkus-spring-di` extension to the `pom.xml` file of your project:
```

Comment:
```suggestion
. Run the following command to add the `quarkus-spring-di` extension to the `pom.xml` file of your project:
```

Reply:
Good point!! Thanks for catching that.  No need to replace it if it's wrong to do so.